src/python/WMCore/JobSplitting/LumiBased.py

```diff
@@ -367,3 +361,39 @@ def algorithm(self, *args, **kwargs):
         self.lumiChecker.closeJob(self.currentJob)
         self.lumiChecker.fixInputFiles()
         return
+
+    def getFilesSortedByLocation(self, lumisPerJob):
```

Comment:
Same comment.

Reply:
This one is different because it does all the work with lumis